source/_components/homekit.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "HomeKit"
+description: "Instructions how to setup the HomeKit component in Home Assistant."
+date: 2018-02-20 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: HomeKit
+ha_release: 0.64
+ha_iot_class:  "Local Polling"
+logo: home-assistant.png
```

Comment:
add the correct logo please

Reply:
I'm not sure if we are allowed to. As far as I know Apple limits the use of it to its certified partners.

Reply:
I think the Homekit logo is OK ( https://www.ehome-news.de/wp-content/uploads/2014/09/homekit.jpg ), we use tons of other logos and it's not the "works with HomeKit"-logo for certified hardware

Reply:
It might sound silly, but I'm not comfortable with uploading it myself, since I don't have the rights for the image. Feel free to upload it and we can add it, otherwise I would leave it at the default logo.